modules/swagger-codegen/src/main/resources/Java/libraries/feign/ApiClient.mustache

```diff
@@ -128,6 +128,11 @@ public class ApiClient {
     ObjectMapper objectMapper = new ObjectMapper();
     objectMapper.enable(SerializationFeature.WRITE_ENUMS_USING_TO_STRING);
     objectMapper.enable(DeserializationFeature.READ_ENUMS_USING_TO_STRING);
+    objectMapper.disable(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES);
+    return objectMapper;
+  }
+
+  public ObjectMapper getObjectMapper(){
```

Comment:
Please add the docstring later when you've time.

PR merged. Thanks again for the contribution.